app/modules/conversations/conversation/conversation_controller.py

```
@@ -114,9 +116,13 @@ async def regenerate_last_message(
         except ConversationServiceError as e:
             raise HTTPException(status_code=500, detail=str(e))
 
-    async def stop_generation(self, conversation_id: str) -> dict:
+    async def stop_generation(
+        self, conversation_id: str, session_id: str = None
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)